rsciio/delmic/_api.py

```diff
@@ -1,80 +1,717 @@
+                        "units": prefix + "s",  # Adjusted unit prefix
+                        "navigate": False,  # No navigation for the T axis
+                    }
+                )
+            elif axis_name == "Angle":
+                # The dimension scale contains the axis in radians, at most between -90° -> 90°, but
+                # can also contain NaN for data acquired outside from the range (typically, irrelevant
+                # data that can be discarded). For now, keep it simple, and just pass the pixel number.
+                scale_a = np.arange(0, image.shape[i])
+                axis_dict.update(
+                    {
+                        "axis": scale_a,
+                        "units": "",
+                        "navigate": False,  # No navigation for the A axis
+                    }
+                )
+            else:  # XYZ
+                # The pixel size is defined in the DimensionScale, as a single value, and the *center* of
+                # the image is defined by the Offset
+                try:
+                    scale_value = float(scale)
+                    prefix = get_unit_prefix(scale_value)
+                except (TypeError, ValueError) as e:
+                    raise TypeError(
+                        f"Expected a numeric value for '{scale_key}', got {type(scale)} instead."
+                    ) from e
+
+                # Y goes up, so top (first) pixel has the largest value => invert scale
+                if axis_name == "Y":
+                    scale_value = -scale_value
+
+                center = float(img_data[offset_key][()])
+                # In HyperSpy, offset is the (center of) the first pixel
+                offset = center - ((image.shape[i] - 1) / 2) * scale_value
+
+                axis_dict.update(
+                    {
+                        "size": image.shape[i],
+                        "offset": offset * unit_factor(prefix),
+                        "scale": scale_value * unit_factor(prefix),
+                        "units": prefix + "m",
+                        "navigate": True,
+                    }
+                )
+        except Exception as ex:
+            _logger.warning("Failed to parse axis %s: %s", axis_name, ex)
+
+    return axes
+
+
+def make_metadata(
+    acq: h5py.Group, acq_type: AcquisitionType, original_md: Dict[str, Any]
+) -> Dict[str, dict]:
+    """
+    Create a metadata dictionary for a given acquisition.
+    See https://hyperspy.org/hyperspy-doc/current/reference/metadata.html
+    In practice, it's used by HyperSpy to instantiate a Signal.
+    """
+    img_title = read_image_title(acq)
+    metadata = {
+        "General": {
+            "title": img_title,
+        },
+        "Signal": {
+            "quantity": "Counts",
+            "signal_type": "",
+        },
+    }
+
+    # If LumiSpy is available => more fancy Signal classes available
+    if lumispy:
```

Comment:
Currently not covered:
https://app.codecov.io/gh/hyperspy/rosettasciio/pull/328?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=hyperspy

But should be easy to test, e.g.:
https://github.com/hyperspy/rosettasciio/blob/f3d59ca640ae169f90864de6ab6970dc32a47a47/rsciio/tests/test_jobinyvon.py#L100